### ISSUE TITLE
Visualizar Classes

### ISSUE THREAD

**Issue by hugo-asb** [MPS-FGA/Grimorio-do-Mestre#9]
Eu, como usuário, desejo visualizar as informações sobre as classes

Critérios de aceitação:
- [ ] Buscar as informações da API (http://www.dnd5eapi.co/docs/#base) 
- [ ] Listar as classes

**Comment by Rdlenke** [comment]
O critério de aceitação "Listar as classes" tá contemplando listar as informações detalhadas de cada classe?

**Comment by hugo-asb** [comment]
Não, deve ser mostrado uma lista com as classes existentes. As Informações detalhadas de cada classe são exibidas quando o usuário clicar em uma dessas que está listada.
